src/MoBi/MoBi.csproj

```diff
@@ -268,8 +268,8 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
-    <PostBuildEvent>call "$(VS140COMNTOOLS)vsvars32.bat"
-editbin /largeaddressaware "$(TargetPath)"</PostBuildEvent>
```

Comment:
@Yuri05  This should not be required anymore correct?

Reply:
correct